share/dss/dss.go

```diff
@@ -208,7 +209,9 @@ func (d *DSS) hashSig() kyber.Scalar {
 	_, _ = d.random.Commitments()[0].MarshalTo(h)
 	_, _ = d.long.Commitments()[0].MarshalTo(h)
 	_, _ = h.Write(d.msg)
-	return d.suite.Scalar().SetBytes(h.Sum(nil))
+	hBuff := h.Sum(nil)
+	encoding.ConvertEndian(hBuff, hBuff)
```

Comment:
a small comment explaining it's because of eddsa verification expecting LE would be worth it I think after all the trouble we had with this one :D